data/meeting_minutes.yml

```diff
@@ -347,6 +347,31 @@ items:
 
 
    specification_committee:
+       -
+          url: specification_committee/minutes-specification-november-4-2020.pdf
+          title: November 4, 2020 (pdf)
+          year: 2020
+       -
+          url: specification_committee/minutes-specification-october-28-2020.pdf
+          title: October 28, 2020 (pdf)
+          year: 2020
+       -
+          url: specification_committee/minutes-specification-october-21-2020.pdf
+          title: October 21, 2020 (pdf)
+          year: 2020
+       -
+          url: specification_committee/minutes-specification-october-14-2020.pdf
+          title: October 17, 2020 (pdf)
+          year: 2020
+       -
+          url: specification_committee/minutes-specification-october-7-2020.pdf
```

Comment:
You have a typo in the actual name of your file. There's an extra space
"minutes-specification-october- 7-2020.pdf"

You will have to update the name of your file.